packages/suite/src/views/settings/device/index.tsx

```diff
@@ -199,13 +217,32 @@ const Settings = () => {
                     <TextColumn
                         title={<Translation id="TR_FIRMWARE_VERSION" />}
                         description={
-                            <>
+                            <Version>
                                 <Translation
                                     id="TR_YOUR_CURRENT_FIRMWARE"
-                                    values={{ version: getFwVersion(device) }}
+                                    values={{
+                                        version: (
+                                            <VersionTooltip content={getFwRevision(device) || ''}>
```

Comment:
assign `const revision = getFwRevision(device);` and use it here and in line 228, fallback is not necessary (?)

Reply:
14d766f